CRM/Core/Payment/AuthorizeNetIPN.php

```diff
@@ -41,13 +41,6 @@ public function __construct($inputData) {
    */
   public function main() {
     try {
-      //we only get invoice num as a key player from payment gateway response.
```

Comment:
no idea what this comment means. Can't see how it adds anything that getting the relevant params & assigning to variables doesn't

ARB translates to 'stuff we handle in this class' & hence is implicit